packages/api/src/schema/api-schema.yaml

```diff
@@ -2138,11 +2544,198 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/error"
+  /asset/request-upload:
+    post:
+      tags:
+        - Asset
+      summary: Upload an asset
+      description: >
+        To upload an asset, your first need to request for a direct upload URL
+        and only then actually upload the contents of the asset.
+
+
+        This also allows your users to upload files since the upload URL is
+        pre-signed with your credentials and is accessible from the browser. The
+        initial API call to get the upload URL must be done from a backend
+        though, from which an API token is available.
+
+
```

Comment:
We should probably remove this paragraph since this API is being used directly from the browser now with CORS keys
```suggestion
```